manifest.json

```diff
@@ -430,6 +430,7 @@
                 "DeleteCollection",
                 "DeleteProject",
                 "DetectFaces",
+                "DetectModerationLabels",
```

Comment:
Could you add an entry in the `src/Service/Rekognition/CHANEGELOG.md` file :pray:

Reply:
Done, there was already a `NOT RELEASED` entry in the changelog so I added it there, let me know if you want me to write a specific version (probably 0.1.8)

Reply:
perfect